episodes/filters-and-thresholding.md

```diff
@@ -228,7 +229,7 @@ viewer.add_labels(mask)
 First, we show a histogram for the image by selecting the 'image' layer, then:  
 `Plugins > napari Matplotlib > Histogram`
 
-![](fig/manual-thresholding-exercise-histogram.png){alt="Histogram of the 
+![](fig/shapes-histogram.png){alt="Histogram of the 
 shape image"}
 
 By moving the left contrast limits node we can figure out what each peak 
```

Comment:
Now that the triangle + square have swapped intensities, the order of the peaks in the text also needs to be updated. Should be from left to right:
- background
- triangle
- circle
- rectangle